pages/index.js

```diff
@@ -412,12 +412,15 @@ export default function Home({ total, today }) {
                     <div className="flow-root rounded-lg bg-gray-50 px-6 pb-8">
                       <div className="-mt-6">
                         <div>
+                        <Link aria-label="Go to feature page"
```

Comment:
This is in a loop, so every Link (I think 6) will all have `Go to feature page`? Not sure that will help with screen readers

Reply:
> This is in a loop, so every Link (I think 6) will all have `Go to feature page`? Not sure that will help with screen readers

we can do `Go to ${feature.name} page`